src/vm-repair/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "0.5.9"
+VERSION = "0.5.10"
```

Comment:
Use version `1.0.0b1` as preview version instead since we should never use `0.x.x` as versioning now. See more details in https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md